package.json

```diff
@@ -5,12 +5,14 @@
   "scripts": {
     "postinstall": "node ./postinstall.js",
     "prepare": "husky install",
-    "lint": "pnpm format-check && eslint --cache .",
+    "lint": "concurrently -c auto --group 'pnpm:lint:*'",
+    "lint:eslint": "eslint --cache .",
+    "lint:prettier": "prettier --cache --list-different '**/*.{js,ts,tsx,md,less,css}'",
+    "lint:tsc": "tsc",
```

Comment:
`concurrently` should suffice for lint scripts until sku has the need for a _real_ task runner.